<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="4">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="1" alignment="Qt::AlignRight">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Rotation parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="ozXlabel">
         <property name="text">
          <string>OZ</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="angleLabel">
         <property name="text">
          <string>Angle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" alignment="Qt::AlignLeft">
        <widget class="QRadioButton" name="oyRot">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="oxLabel">
         <property name="text">
          <string>OX</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" alignment="Qt::AlignLeft">
        <widget class="QRadioButton" name="ozRot">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2" alignment="Qt::AlignLeft">
        <widget class="QRadioButton" name="oxRot">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="oyLabel">
         <property name="text">
          <string>OY</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="angle"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1" alignment="Qt::AlignRight">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Translation parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="oxLabel1">
           <property name="text">
            <string>OX</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="oxTrans"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="oyLabel1">
           <property name="text">
            <string>OY</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="oyTrans"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="ozLabel1">
           <property name="text">
            <string>OZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ozTrans"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>39</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="apply">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
